flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ArchivedExecutionGraph.java

```diff
@@ -113,6 +113,10 @@ public class ArchivedExecutionGraph implements AccessExecutionGraph, Serializabl
 
     @Nullable private final String changelogStorageName;
 
+    @Nullable private final String streamGraphJson;
```

Comment:
The nullable annotation should be removed, as by design, the stream graph JSON will not be null.

Reply:
Thanks for the reminder. Removed.